SatChecker.py

```diff
@@ -643,6 +650,8 @@ def __build_cycle(self, reset, cycle):
 
     def __build_formula(self):
         self.__find_reset(self.rst_phase)
+        self.__wait_init_delay()
+
         cycle = 0
```

Comment:
We should potentially set the cycle count to self.init_delay here, so that it avoids confusion, (also interacts with a bunch of other options like --from-cycle)